packit_service/service/static/style.css

```diff
@@ -2,7 +2,9 @@
   font-family: sans-serif;
 }
 
-pre {
+pre,
+code,
+kbd {
```

Comment:
Wow, HTML really turned fancy :smile:  

How about adding `samp` and `var`, too? Just in case :)

Reply:
I'd be thumbs up for Latin Modern for `var` :smirk: :joy: gonna add them